deploy/charts/harvester-crd/README.md

```diff
@@ -0,0 +1,63 @@
+# Harvester CRD Helm Chart
+
+## Chart Details
+
+This chart includes following CRDs:
+
+- Harvester (Automatically generated by [this script](https://github.com/harvester/harvester/blob/master/scripts/generate-manifest))
+- KubeVirt
+- csi-snapshotter
+- whereabouts
+
+## Introduction
+
+Harvester uses a `ManagedChart` resource to deploy the [Harvester CRDs](https://github.com/harvester/harvester/tree/master/deploy/charts/harvester-crd). This is configured via the [harvester-installer](https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-10-harvester.yaml). Be aware that `deploy/harvester` and `deploy/harvester-crd` are two separate `ManagedChart` resources.
+
+Harvester depends on several components that provide the CRDs it needs. However, most of these CRDs must be created manually. To simplify management, Harvester uses an [independent folder to manage all CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-2-separate-charts) to deploy with Helm.
+
+That said, not every CRD can be managed by Helm. Take `cdi` CRD as an exmaple, installing `cdi-operator` doesn't create the CRD for us. However, the `cdi-operator` will [remove obsolete version from existing `cdi` CRD](https://github.com/kubevirt/containerized-data-importer/blob/e136558c999acce03291849aacc8154026e58445/pkg/operator/controller/callbacks.go#L61-L62). This is also mentioned in [#7912 (comment)](https://github.com/harvester/harvester/issues/7912#issuecomment-2743944344).
+
+
+Therefore, managing it with Helm is not ideal. Hence, Harvester creates it manually. These CRDs are applied in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L1358) script.
+
+Additionally, Longhorn CRDs are not managed here either because Longhorn handles its own CRD creation process.
```

Comment:
```suggestion
Harvester simplifies the upgrade of the CRDs by orchestrating their manifests in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L1358) script. This ensures that users are not burdened by the varying schema upgrade strategies of the different dependency projects, such as the built-in schema removal mechanism in [CDI](https://github.com/kubevirt/containerized-data-importer/blob/e136558c999acce03291849aacc8154026e58445/pkg/operator/controller/callbacks.go#L61-L62).

::: note

 Longhorn CRDs are not managed here because Longhorn handles its own CRD creation process.

:::
```